ports/OpenBSD/VictoriaMetrics/Makefile

```diff
@@ -4,9 +4,9 @@ COMMENT =		fast, cost-effective and scalable time series database
 
 GH_ACCOUNT =		VictoriaMetrics
 GH_PROJECT =		VictoriaMetrics
-GH_TAGNAME =		v1.46.0
+GH_TAGNAME =		v1.58.0
```

Comment:
It would be better updating to v1.59.0 - see https://github.com/VictoriaMetrics/VictoriaMetrics/releases

Reply:
I missed this one :)